servo/connector.py

```diff
@@ -135,10 +135,25 @@ class BaseConnector(
     )
     """Shared configuration from our parent Servo instance."""
 
+    _optimizer: Optional[
+        Union[
+            servo.configuration.OpsaniOptimizer,
```

Comment:
I was somewhat apprehensive about type aliases due to their display implementation in VSCode; specifically that its easier to mistake them for a wholly distinct type rather than shorthand for a combination of existing types. You have to navigate to the field definition in order to see that its an alias and searching the codebase for an alias in the wrong way (eg `class OptimizerTypes`) may cause confusion by appearing to not exist (as happened to me in the past with servo.api.Statuses).

This is all to say I agree being less verbose is ideal but wanted to ensure you were aware of this potential pitfall.

https://github.com/microsoft/pylance-release/discussions/2715